docs/general/deployment.md

```diff
@@ -137,20 +137,16 @@ _Step 3:_ Within the root of your react-boilerplate source folder, execute the f
 
 ## Netlify
 
-Netlify is a static site host that deploys from Git repos. To deploy `react-boilerplate` from Netlify:
+Netlify is a static site host that deploys from Git repos. To deploy a `react-boilerplate` app from Netlify, you can leverage their continuous deployment feature. All you have to do is push to your repo and Netlify will [run your production build script](https://docs.netlify.com/configure-builds/get-started/).
 
-_Step 1:_ Clone `react-boilerplate` and remove the default project (`npm run clean`)
+_Step 1:_ Clone react-boilerplate and remove the default project (`npm run clean`)
```

Comment:
Can we improve this step with something like this which points to the quick start instructions?

_Step 1:_ Follow react-boilerplate's [quick start instructions](https://github.com/react-boilerplate/react-boilerplate#quick-start)